src/testshade/testshade.cpp

```diff
@@ -159,6 +189,18 @@ set_shadingsys_options ()
     shadingsys->attribute ("userdata_isconnected", userdata_isconnected);
     if (! shaderpath.empty())
         shadingsys->attribute ("searchpath:shader", shaderpath);
+    if (librarypath.empty()) {
+        // Look in expected dist directory tree,
+        // also look in expected build tree for the unittests
+        // TODO: test on windows
+#if (defined(_WIN32) || defined(_WIN64))
+        static const char * relative_lib_dir = "\\..\\lib64:\\..\\liboslexec";
+#else
+        static const char * relative_lib_dir = "/../lib64:/../liboslexec";
+#endif
+        librarypath = get_executable_directory() + relative_lib_dir;
```

Comment:
Looking at this line now, looks like the 1st relative_lib_dir entry gets the parent directory appended.  I'll have to update this to a loop to build them.

Reply:
```
#if (defined(_WIN32) || defined(_WIN64))
        static const char * relative_lib_dirs[] = {"\\..\\lib64", "\\..\\lib"};
#else
        static const char * relative_lib_dirs[] = {"/../lib64", "/../lib"};
#endif
        auto executable_directory = OIIO::Filesystem::parent_path(OIIO::Sysutil::this_program_path());
        bool dirNum = 0;
        for(const char * relative_lib_dir:relative_lib_dirs) {
            if(dirNum++ > 0) 
                librarypath	+= ":";
            librarypath += executable_directory + relative_lib_dir;
        }
```
produces what I expect:
```
        librarypath = /nfs/pdx/home/amwells/OSL_Dev/github/OpenShadingLanguage/dist/linux64.profile/bin/../lib64:/nfs/pdx/home/amwells/OSL_Dev/github/OpenShadingLanguage/dist/linux64.profile/bin/../lib
```

Reply:
great